arb/test/t-dump-file.c

```diff
@@ -0,0 +1,81 @@
+/*
+    Copyright (C) 2019 Julian Rüth
+
+    This file is part of Arb.
+
+    Arb is free software: you can redistribute it and/or modify it under
+    the terms of the GNU Lesser General Public License (LGPL) as published
+    by the Free Software Foundation; either version 2.1 of the License, or
+    (at your option) any later version.  See <http://www.gnu.org/licenses/>.
+*/
+
+#include <string.h>
+#include <stdio.h>
+#include "arb.h"
+
+int main()
```

Comment:
Tests are essentially six times the same stuff repeated (arb, arf, mag × str, FILE). Is that how this is supposed to be?

Reply:
Sure.